zerver/tests/test_message_edit.py

```diff
@@ -851,36 +855,34 @@ def do_edit_message_assert_error(id_: int, unique_str: str, error: str) -> None:
 
         # any user can edit the topic of a message
         set_message_editing_params(True, 0, Realm.EDIT_TOPIC_POLICY_EVERYONE)
-        # log in as a new user
-        self.login("cordelia")
-        do_edit_message_assert_success(id_, "A")
+        do_edit_message_assert_success(id_, "A", "cordelia")
```

Comment:
These assertions feel a bit hard to read; should we make some of these parameters keyword-only arguments (https://www.python.org/dev/peps/pep-3102/)?

Reply:
Yes we can do that. One thing that I thought was can we directly pass the new topic name like "topicA", "topicB" itself here rather than appending the old topic with a character. I think this will make this readable with keyword-only arguments - `message_id`, `new_topic` and `acting_user`.

Reply:
Yeah, this is a good idea.